lang/js/suggested-app-structure.md

```diff
@@ -0,0 +1,27 @@
+Suggested App Structure
+=======================
+
+The below is an example application structure. This application uses HTTP as a communication layer but it would work equally well with other communication types.
```

Comment:
Don't need `The` at the beginning of the sentence.

Maybe put `example` instead of `application` in the second sentence.